### DISCUSSION TITLE
Basic setup of Webpack with React App. ---  font not showing.

### DISCUSSION THREAD

**Discussion by Elindo586** [webpack/webpack#15768] [Q&A]
So, I am trying to use webpack to do SRR with REACT.

You can see the entire code here:
https://github.com/Elindo586/myWebpackTest

Right now I got no errors showing, but the localhost isn't showing any text, however, it is showing the css for the background? Go figure..

Is there any links to follow a basic setup for REACT?   Maybe I am missing a plugin? a loader? 

Following the REACT regular set up, and the webpack-config.js  the main files that I think are in play look like this:

Index.html
```
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>Document</title>
  </head>
  <body>
    <h1 id="app"></h1>
  </body>
</html>
```

App.js
```
import "../styles.css";
import "../styles.scss";

function App() {
  return (
    <div className="App">
      <h1>Hello!</h1>
    </div>
  );
}

export default App;
```
index.js
```
import React from "react";
import ReactDOM from "react-dom";
import App from "./components/App";

ReactDOM.render(<App />, document.getElementById("app"));
```

**Comment by vankop** [comment]
you can use [`create-react-app`](https://create-react-app.dev/) this tool is on top of webpack. Also it allows to "eject" CRA (create-react-app) by running https://create-react-app.dev/docs/available-scripts/#npm-run-eject

**Reply by Elindo586** [reply]
Oh... I already have react installed.

`npm install react react-dom `

I just don't know why the text isn't showing.

Not sure if it is because I am missing something from webpack, or for something else.

**Comment by Elindo586** [comment]
I had to import React to the App.js

This is working different than the regular REACT app set up. 




```
import React from "react";
import ReactDOM from "react-dom/client";

import "../styles.css";
import "../styles.scss";

function App() {
  return (
    <div className="App">
      <h1>Hello!</h1>
    </div>
  );
}

export default App;
```
